cmsdb/processes/ewk.py

```diff
@@ -209,15 +217,29 @@
 
 # NNLO cross section, based on:
 # https://twiki.cern.ch/twiki/bin/view/CMS/StandardModelCrossSectionsat13TeV?rev=27
+# and for 13.6 TeV, based on:
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
+
+wm_lnu_xs_13p6 = const.n_leps * Number(9009.5, {
+    "scale": (0.014j, 0.012j),
+    "pdf": 0.008j,
+})
+wp_lnu_xs_13p6 = const.n_leps * Number(12122.5, {
+    "scale": (0.011j, 0.014),
+    "pdf": 0.007j,
+})
 
 w_lnu = w.add_process(
     name="w_lnu",
     id=6100,
     label=rf"{w.label} ($W \rightarrow l\nu$)",
-    xsecs={13: const.n_leps * Number(20508.9, {
-        "scale": (165.7, 88.2),
-        "pdf": 770.9,
-    })},
+    xsecs={
+        13: const.n_leps * Number(20508.9, {
+            "scale": (165.7, 88.2),
+            "pdf": 770.9,
+        }),
+        13.6: wm_lnu_xs_13p6 + wp_lnu_xs_13p6,
```

Comment:
```suggestion
        13.6: wm_lnu_xs_13p6 + wp_lnu_xs_13p6,  # addition necessary due to absence of combined value, but values at 13 TeV in https://twiki.cern.ch/twiki/bin/viewauth/CMS/StandardModelCrossSectionsat13TeV?rev=28 show that there might be a permille error in this calculation
```

Doing a simple addition does not work (error 1 per mille) for the values at 13TeV in https://twiki.cern.ch/twiki/bin/viewauth/CMS/StandardModelCrossSectionsat13TeV?rev=28. In my opinion, you should add a comment saying that this is the case and you have no other option since no combined value is given

Reply:
I am not really sure if there is really an error due to adding the two (W+ and W-) cross sections, it's just two different calculations to get to the same result and the difference between the two results is covered by the uncertainties.

Reply:
Yes, I am not sure either and it might be a simple difference in calculation, I am just wondering if am missing something allowing correlations between these two values... Which is why I proposed a disclaimer. I would also be fine without if you are sure there are no correlations